.tool-versions

```diff
@@ -0,0 +1,3 @@
+nodejs 21.2.0
+ruby 3.3.5
+cocoapods 1.16.2
```

Comment:
`nodejs 21.2.0` pulled from `macos/.xcode.env.local`. Should probably be upgraded since this is EOL but not a big deal atm 🤷

`ruby 3.3.5` is closest release to this version of `cocoapods`

`cocoapods 1.16.2` pulled from the end of `macos/Podfile.lock`